test/new-e2e/tests/installer/linux_test.go

```diff
@@ -54,23 +54,23 @@ func runTest(t *testing.T, pkgManager string, arch os.Architecture, distro os.De
 }
 
 func TestCentOSARM(t *testing.T) {
-	// t.Parallel()
+	t.Parallel()
```

Comment:
<!-- Warning highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes and they are blockers. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question. -->
**🔨 warning**
Please don't use `t.Parallel` with e2e tests, it would OOM the runner

Reply:
I don't have a strong opinion so we can remove it in a subsequent PR, but how would we OOM? While we run the test suites in parallel, this seems fine because:
- Each test suite is on its own VM
- For a single test suite, tests are ran sequentially